.github/actions/test-driver/action.yml

```diff
@@ -13,54 +13,21 @@ inputs:
     required: false
     description: Suffix to append to the test logs artifact name to avoid conflicts in matrix jobs
 
-outputs:
-  quarantine-skipped:
-    description: Whether the driver test was skipped due to quarantine
-    value: ${{ steps.check-driver.outputs.skip }}
-
 runs:
   using: "composite"
   steps:
-    - name: Check if driver is ignored
-      id: check-driver
-      shell: bash
-      run: |
-        # If DRIVERS env var is not set, proceed with tests
-        if [ -z "$DRIVERS" ]; then
-          echo "No DRIVERS env var set. Proceeding with tests."
-          echo "skip=false" >> $GITHUB_OUTPUT
-          exit 0
-        fi
-
-        # Fetch CI config and check if driver should be skipped
```

Comment:
This check happens inside of `mage -driver-decisions` now. So instead of starting the job, and each one checking, we check it once, and use the `if`s to skip starting jobs in the first place.